configurationdemo/source/c/src/kaa_demo.c

```diff
@@ -85,28 +75,41 @@ int main(/*int argc, char *argv[]*/) {
 
     if (error_code) {
         demo_printf("Failed to add configuration receiver\r\n");
-        return 3;
+        return 1;
     }
 
+    /*
+     * Display default configuration.
+     */
     kaa_demo_print_configuration_message(
             kaa_configuration_manager_get_configuration(
                     kaa_client_get_context(kaa_client)->configuration_manager));
 
+    /*
+     * Obtain and display Endpoint Key Hash.
+     */
+    uint8_t *endpoint_key_hash = NULL;
+    size_t endpoint_key_hash_length = 0;
+
+    ext_get_sha1_base64_public(&endpoint_key_hash, &endpoint_key_hash_length);
+
+    printf("Endpoint Key Hash: %.*s\r\n", (int)endpoint_key_hash_length,
+            endpoint_key_hash);
+
     /**
      * Start Kaa client main loop.
      */
     error_code = kaa_client_start(kaa_client, NULL, NULL, 0);
     if(error_code) {
         demo_printf("Failed to start Kaa main loop\r\n");
-        return 4;
+        return 1;
```

Comment:
If you prefer return same values in all error cases, consider using `EXIT_SUCCESS` and `EXIT_FAILURE` values.